### ISSUE TITLE
Crashes on Library update

### ISSUE THREAD

**Issue by dm2912** [jurialmunkey/plugin.video.themoviedb.helper#364]
Discovered earlier looking in settings that my TMDB helper hasnt succesfully completed a library update since the 7th.

Just ran update a few times and each time crashed:

Same errors in the log each time and the library update stops on that error

```
 The Challenge (TMDB 62)
                                            Previously Added Season 33. Skipping...
2020-11-18 00:30:57.484 T:3087012736  NOTICE: [plugin.video.themoviedb.helper]
                                            The Challenge (TMDB 62)
                                            Previously Added Season 34. Skipping...
2020-11-18 00:30:57.756 T:3087012736  NOTICE: [plugin.video.themoviedb.helper]
                                            The Challenge (TMDB 62) - Season 35 - Done!
                                            Skipped Previously Added Episodes:
                                            ['S35E01 - Mad World', 'S35E02 - Sweet Dreams Are Made of Dee', 'S35E03 - A Hard Jays Night', 'S35E04 - Karma Chameleon', 'S35E05 - Break Up With Your BF Im Bored', 'S35E06 - Love Will Tear Us Apart', 'S35E07 - Should I Stay Or Should I Go', 'S35E08 - Let Live and Die', 'S35E09 - Backstabber', 'S35E10 - The Final Countdown', 'S35E11 - You Aint Right', 'S35E12 - Pictures of You', 'S35E13 - Victim of Love', 'S35E14 - Your Time is Gonna Come', 'S35E15 - Crash Into Me', 'S35E16 - Its the End of the World As We Know It', 'S35E17 - Reunion', 'S35E18 - Behind total madness']
2020-11-18 00:30:57.995 T:3087012736   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.TypeError'>
                                            Error Contents: normalize() argument 2 must be unicode, not None
                                            Traceback (most recent call last):
                                              File "/storage/.kodi/addons/plugin.video.themoviedb.helper/script.py", line 8, in <module>
                                                Script().router()
                                              File "/storage/.kodi/addons/plugin.video.themoviedb.helper/resources/lib/script/router.py", line 312, in router
                                                return library_update(**self.params)
                                              File "/storage/.kodi/addons/plugin.video.themoviedb.helper/resources/lib/script/router.py", line 233, in library_update
                                                force=kwargs.get('force', False))
                                              File "/storage/.kodi/addons/plugin.video.themoviedb.helper/resources/lib/kodi/update.py", line 484, in library_autoupdate
                                                add_userlist(user_slug, list_slug, confirm=False, allow_update=False, busy_spinner=busy_spinner, force=force)
                                              File "/storage/.kodi/addons/plugin.video.themoviedb.helper/resources/lib/kodi/update.py", line 418, in add_userlist
                                                tmdb_id=tmdb_id, imdb_id=imdb_id, tvdb_id=tvdb_id, p_dialog=p_dialog, force=force)
                                              File "/storage/.kodi/addons/plugin.video.themoviedb.helper/resources/lib/kodi/update.py", line 289, in add_tvshow
                                                validify_filename(episode.get('name')))
                                              File "/storage/.kodi/addons/plugin.video.themoviedb.helper/resources/lib/files/utils.py", line 29, in validify_filename
                                                filename = str(unicodedata.normalize('NFD', filename).encode('ascii', 'ignore').decode("utf-8"))
                                            TypeError: normalize() argument 2 must be unicode, not None
                                            -->End of Python script error report<--
2020-11-18 00:36:32.061 T:3598001024   ERROR: Traceback (most recent call last):
```

can get full logs if need be.

This is on the latest Coreelec, and the latest repo version of the addon (clean install and settings wipe of tmdb helper prior to last run of the library update)

**Comment by dm2912** [comment]
note ive just updated from master and will test with that as I see you just made a change to router.py

**Comment by jurialmunkey** [comment]
Yeah I think I already fixed this issue. I made major changes to library updater in the past few days and these log messages look like they're from the old version.

The new version doesn't even output library update messages to the log at all. Instead it creates a separate log specifically of what it updated in the addon_data folder:
`.kodi/userdata/addon_data/plugin.video.themoviedb.helper/log_library/yyyy-mm-dd-HHMMSS.json`
